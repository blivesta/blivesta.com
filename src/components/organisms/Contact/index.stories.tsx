import React from 'react';
// import styled from 'styled-components';
import { storiesOf } from '@storybook/react';
import Contact from '.';

storiesOf(`organisms|Contact`, module).add(`default`, () => <Contact />);
