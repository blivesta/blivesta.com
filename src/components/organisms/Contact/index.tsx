import * as React from 'react';

import Title from '../../atoms/Title';
import P from '../../atoms/P';
import ContentBlock from '../../molecules/ContentBlock';
import NetlifyForm from '../NetlifyForm';

const Contact = () => {
  return (
    <ContentBlock id="contact" title="Contact">
      <Title>Let’s talk.</Title>
      <P lang="en">
        Please share your questions, comments and business inquiries below and we will get back to you as soon as
        possible.
      </P>
      <P lang="ja" last>
        ご質問、ご意見、ビジネスに関するお問い合わせは下記にご記入ください。できるだけ早くご連絡いたします。
      </P>
      <NetlifyForm />
    </ContentBlock>
  );
};

export default Contact;
