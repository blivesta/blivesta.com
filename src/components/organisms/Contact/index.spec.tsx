import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Contact from '.';

describe('Contact', () => {
  it('snapshot', () => {
    const tree = shallow(<Contact />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
