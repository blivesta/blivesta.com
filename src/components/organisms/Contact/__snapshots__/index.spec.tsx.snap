// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact snapshot 1`] = `
<ContentBlock
  id="contact"
  title="Contact"
>
  <styled.h1>
    Let’s talk.
  </styled.h1>
  <P
    lang="en"
  >
    Please share your questions, comments and business inquiries below and we will get back to you as soon as possible.
  </P>
  <P
    lang="ja"
    last={true}
  >
    ご質問、ご意見、ビジネスに関するお問い合わせは下記にご記入ください。できるだけ早くご連絡いたします。
  </P>
  <NetlifyForm />
</ContentBlock>
`;
